openedx/features/discussions_plugin_piazza/views.py

```diff
@@ -0,0 +1,45 @@
+from lti_consumer.lti_1p1.contrib.django import lti_embed
+from opaque_keys.edx.keys import CourseKey
+from web_fragments.fragment import Fragment
+
+from lms.djangoapps.discussion.api.config import get_discussion_config
```

Comment:
Another invalid import. 
```suggestion
from openedx.core.djangoapps.discussions.api.config import get_discussion_config
```

Reply:
Resolved in next commit